src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -347,9 +353,11 @@ static BOOL redirectStdStreams(APX_STDWRAP *lpWrapper, LPAPXCMDLINE lpCmdline)
         if ((lpWrapper->fpStdOutFile = _wfsopen(lpWrapper->szStdOutFilename,
                                                L"a",
                                                _SH_DENYNO))) {
-            _dup2(_fileno(lpWrapper->fpStdOutFile), 1);
-            *stdout = *lpWrapper->fpStdOutFile;
+            int ret = _dup2(_fileno(lpWrapper->fpStdOutFile), (_fileno)(stdout));
+            if (ret == -1)
+                apxLogWrite(APXLOG_MARK_ERROR "redirectStdStreams _dup2 failed on stdout");
```

Comment:
How about using "redirectStdStreams()" instead of "redirectStdStreams" to make it obvious the context is within a function?

Reply:
fixed